library/std/src/intrinsic.qs

```diff
@@ -368,17 +368,12 @@ namespace Microsoft.Quantum.Intrinsic {
     /// The output result is given by the distribution:
     /// $$
     /// \begin{align}
-    ///     \Pr(\texttt{Zero} | \ket{\psi}) =
```

Comment:
I think this line got dropped, so the rewrite is not just replacing boldone with I, it changes the formula? And it doesn't look like it should be the breaking one, worst case scenario we can replace \Pr with text

Reply:
Some time ago we decided that \Pr(\texttt{Zero} | \ket{\psi}) is not a standard notation so we left this out of the formula, but we should say this in the text I was under impression that we fixed this somewhere. Maybe there's another place where it is already fixed, and we can take text from there.

Reply:
Looks standard to me, probability of measuring zero when measuring \psi?
But the new expression alone is not a clear definition of distribution, some edit to text might help

Reply:
OK, I found the fixed version. Right above this variant of the function there's another one for @Config(QubitReset). That one is where description has been fixed. We should reuse that description (and probably the entire doc comment.
This also brings an interesting question - different variants of functions for different capabilities may reuse doc comments, but we don't support such reuse.

Reply:
I've copied over the description from the @config(QubitReset) version, as suggested, with the minor change of renaming that section from "Description" to "Remarks" to fit the format of the other doc strings in the file.